e2e/cypress/tests/integration/enterprise/cloud/billing/cloud_pricing_modal_spec.js

```diff
@@ -272,8 +272,8 @@ describe('Pricing modal', () => {
         // *Check that starter Downgrade card  button exists
         cy.get('#pricingModal').get('#starter').get('#starter_action').contains('Downgrade');
 
-        // *Check that professsional card Upgrade button is disabled while on enterprise
-        cy.get('#pricingModal').get('#professional').get('#professional_action').should('be.disabled');
```

Comment:
Can we keep the case of disabled if they are indeed in Enterprise (no trial)?

Reply:
That test case is below on line 434